<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>521</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>18</red>
          <green>14</green>
          <blue>6</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>176</red>
          <green>233</green>
          <blue>230</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>47</green>
          <blue>48</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>176</red>
          <green>233</green>
          <blue>230</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Polish calculator</string>
     </attribute>
     <widget class="QLineEdit" name="expression">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
     </widget>
     <widget class="QPushButton" name="EqualButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>430</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 1px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphX" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RESULT:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Result_label">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ValueX">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>maxX =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="MaxX">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>minX =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="MinX">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="MaxY">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="MinY">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>maxY =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>21</red>
            <green>17</green>
            <blue>7</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>18</red>
            <green>14</green>
            <blue>6</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>208</red>
            <green>208</green>
            <blue>208</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>208</red>
            <green>208</green>
            <blue>208</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="96">
            <red>164</red>
            <green>166</green>
            <blue>168</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="96">
            <red>164</red>
            <green>166</green>
            <blue>168</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>minY =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>170</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>step = </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="step">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CreditCalculator">
     <attribute name="title">
      <string>Credit Calculator</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Сумма вклада:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Срок:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Процентная ставка:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Тип:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Summ">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Date">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Procent">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="type">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Аннуитетный</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дифференцированный</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="count">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 1px;
}</string>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="year">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>103</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Лет</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Месяцев</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Ежемесячный платеж:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Переплата по кредиту:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Общая выплата:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Monthplate">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pereplata">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="All_money">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Deposit calculator</string>
     </attribute>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Сумма вклада:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Summ_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Процентная ставка:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Summ_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Срок размещения:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="count_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>320</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: blue;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 1px;
}</string>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Date_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="year_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>103</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Дней</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Месяцев</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Лет</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Начало срока:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="year_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Фиксированная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Зависит от суммы</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Зависит от срока</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Переодичность выплат:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="year_5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Каждый день</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Каждую неделю</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в месяц</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в квартал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в полгода</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в год</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>В конце срока</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Капитализация процентов</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Пополнения:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="year_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Разовое</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в месяц</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в 2 месяца</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в квартал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в полгода</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в год</string>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Date_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Сумма"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Частичные снятия:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="year_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Разовое</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в месяц</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в 2 месяца</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в квартал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в полгода</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Раз в год</string>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Date_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string extracomment="Сумма"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Начисленные проценты:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Налог:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>410</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>Сумма на вкладе к концу срока:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
